<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewTeacherDialog</class>
 <widget class="QDialog" name="AddNewTeacherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Teacher</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>541</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Personal</string>
    </attribute>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FName</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MName</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mobile</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Department</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineName_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>21</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineFName_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMName_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineAddress_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="linePhone_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMobile_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEmail_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineDepartment_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Birth Date</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>271</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Courses</string>
    </attribute>
    <widget class="QListWidget" name="listAllCourses">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>501</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose a department</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDeps">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableCanCourses">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>501</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Languages</string>
    </attribute>
    <widget class="QListWidget" name="listCanDiplomas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>491</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose a language</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboSprachen">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listAllDiplomas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>491</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Salary</string>
    </attribute>
    <widget class="QTableView" name="tableSalary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>511</width>
       <height>321</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pick a salary</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Kassen</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboKassen">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>420</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>Unavail</string>
    </attribute>
    <widget class="QTableView" name="tableUnAvail">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushSaveTeacher">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>450</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
